Kickstarter-iOS/Features/PledgedProjectsOverview/CardView/PPOProjectCardModel.swift

```diff
@@ -340,6 +346,9 @@ extension PPOProjectCardModel {
     let primaryAction: PPOProjectCardModel.Action
     let secondaryAction: PPOProjectCardModel.Action?
     let tierType: PPOProjectCardModel.TierType
+    // For v1 of PPO we're just using the same url for surveys and the backing details page.
+    // This specifically links to the survey tab.
+    let backingDetailsUrl = backing?.backingDetailsPageRoute
```

Comment:
Sanity checking, is there any validation that needs to be done to ensure this string is ok to use? It's fine if not, but if there is it should be done here.

Reply:
I don't think so. We get the string straight from our backend and worse case scenario we'd open the survey web view controller to a blank page (which isn't great but I see no reason why it would ever happen). We could confirm that it's not the empty string if you want?